scripts/benchmark/go.sh

```diff
@@ -0,0 +1,24 @@
+#!/usr/bin/env bash
+
+commits=("${@}")
+
+function cleanup {
+  echo Cleaning up...
+  for commit in "${commits[@]}"; do
+    rm -rf $commit
+  done
+  echo Done!
+}
+trap cleanup EXIT
+
+args=()
+
+for commit in "${commits[@]}"; do
+  rm -rf $commit
+  git -C ../.. archive $commit --prefix $commit/ | tar -x
+  (cd $commit; yarn; yarn build)
```

Comment:
1. Use a temp dir to build, and copy `dist` to `dist-${commit}`
2. Run test from project root,  instead of cd to this dir

WDYT?

Reply:
1. I excluded the tests directories from the `git archive` command, so it became more lightweight. I don't see a point in making this script more complicated.
2. I added that. Now it can be run from anywhere inside the repo.